geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -1484,10 +1484,10 @@ def themes(self):
                                 if 'alias' in c.attrib:
                                     attrib[c.attrib['name']] = c.attrib['alias']
                             types[child.attrib['name']] = attrib
-
                     attributes = {}
                     for name, type_ in elements.items():
-                        attributes[name] = types[type_]
+                        if type_ in types:
+                            attributes[name] = types[type_]
```

Comment:
I think that we should add a warning message ...

Reply:
in the console?

Reply:
Yes :-)

Reply:
In the else :-)